driver/normalizer/annotation.go

```diff
@@ -1,277 +1,448 @@
 package normalizer
 
 import (
-	"errors"
+	"go/token"
 
-	"github.com/bblfsh/go-driver/driver/goast"
 	"gopkg.in/bblfsh/sdk.v1/uast"
-	. "gopkg.in/bblfsh/sdk.v1/uast/ann"
-	"gopkg.in/bblfsh/sdk.v1/uast/transformer"
-	"gopkg.in/bblfsh/sdk.v1/uast/transformer/annotatter"
+	"gopkg.in/bblfsh/sdk.v1/uast/role"
+	. "gopkg.in/bblfsh/sdk.v1/uast/transformer"
 	"gopkg.in/bblfsh/sdk.v1/uast/transformer/positioner"
 )
 
-// Transformers is the of list `transformer.Transfomer` to apply to a UAST, to
-// learn more about the Transformers and the available ones take a look to:
+// Native is the of list `transformer.Transformer` to apply to a native AST.
+// To learn more about the Transformers and the available ones take a look to:
 // https://godoc.org/gopkg.in/bblfsh/sdk.v1/uast/transformer
-var Transformers = []transformer.Tranformer{
-	annotatter.NewAnnotatter(AnnotationRulesExpr),
-	annotatter.NewAnnotatter(AnnotationRulesStmt),
-	annotatter.NewAnnotatter(AnnotationRules),
+var Native = Transformers([][]Transformer{
+	// The main block of transformation rules.
+	{Mappings(Annotations...)},
+	{
+		Mappings(
+			AnnotateIfNoRoles("FieldList", role.Incomplete),
+			Unannotated(),
```

Comment:
The Unannotated role on the other side should be applied automatically. Having do add a Transformer in the driver breaks a little BIP-2 since it should be automatically done despite whatever the driver programmer does, but I guess we can work with this until we find a way to apply it automatically.

(same for the other drivers).